controllers/emailController.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     let hash = crypto.createHash('md5').update(req.body.to).digest('hex')
     var guest = { guestEmail: req.body.to, emailed: true, emailHash: hash }
     req.body.message = req.body.message.replace(/\n/ig, '<br>')
+    console.log(req.body.url)
```

Comment:
req.body.url is returning undefined here and in the URL. Do I need to merge anything else?

Reply:
Nvm, I updated the .env